parity-clib-python-example/include/parity.h

```diff
@@ -0,0 +1,105 @@
+// Copyright 2018 Parity Technologies (UK) Ltd.
```

Comment:
for some reason, this only worked when I patched this file to use a c typedef struct instead of the regular struct. Apparently, the type ParityParams requires it to be a typedef? I'm just a bit confused as well, can we use a struct as a type without marking it with typedef?

Thanks so much for all your help